### DISCUSSION TITLE
Library - Alphabet Bar Bug

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#507] [Fixed Issues]
I find that if I use the alphabet bar to locate an artist, then click on the artist name, only the last album in the folder is shown. I have to swipe down to see the other albums.

**Comment by ceejayemm** [comment]
Seems OK on 20210911 (32 bit) on Win 10 / MS Edge. I presume this relates to the mobile app ?

Chris

**Comment by R101G** [comment]
@ceejayemm  No, it happens with Firefox on a desktop, as well as Chrome on a tablet. I don't think there is an app for rAudio.

rAudio 1 • 20210911
5.10.63-1-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Comment by ceejayemm** [comment]
R101-Github

The RuneAudio app (https://play.google.com/store/apps/details?id=com.runeaudio&hl=en_GB&gl=US) for Android still works for rAudio1 if its of interest. You have to manually enter the IP number as it won't auto discover it though.

Back to your issue, I am using 20210911 on a PI Zero and PI 3B without any problem in this respect:

rAudio 1 • 20210911
5.10.39-2-ARCH armv6l
Raspberry Pi Zero W Rev 1.1
BCM2835 • 455MiB
ARM1176 @ 1GHz

and 

rAudio 1 • 20210911
5.10.20-1-ARCH armv7l
Raspberry Pi 3 Model B Rev 1.2
BCM2837 • 962MiB
4 x Cortex-A53 @ 1.2GHz

As noted earlier MS-Edge seems OK and I have just tried Chrome (93.0.4577.82 (Official Build) (64-bit)) on my desktop and that seems OK too - see attached screenshots.

[chrome.zip](https://github.com/rern/rAudio-1/files/7160159/chrome.zip)


Chris

**Comment by R101G** [comment]
@ceejayemm If I had clicked G on your library list, and then clicked on the artist, only the We Can't Dance album would be shown until I scrolled up. The exception would be if the library list was so short that the Genesis folder was already displayed before I clicked G. The initial list of artist folders has to move up for the bug to show. I think the "move up" action may somehow be carried over into the folders.

**Comment by R101G** [comment]
Fixed by 20210924 update.
